cmd/noms/noms_log.go

```diff
@@ -99,8 +100,12 @@ func runLog(args []string) int {
 	defer pgr.Stop()
 
 	for commitBuff := range outChan {
-		io.Copy(pgr.Writer, bytes.NewReader(commitBuff.([]byte)))
+		_, err := io.Copy(pgr.Writer, bytes.NewReader(commitBuff.([]byte)))
+		if err != nil {
+			os.Exit(0)
```

Comment:
I've totally run into this problem before, or something very similar to it. I don't remember when, but it probably was either noms show, diff, or log. @cmasone-attic might be interested.